GameDefinition/PLAYING.md

```diff
@@ -431,10 +431,22 @@ at least 5% of damage always gets through, even if the bonus is nominally
 below -95%, so excessively strong armor acts only as a buffer against
 high melee skill of opponents.
 
-The current pointman's melee bonus, armor modifier and other detailed
-stats can be viewed in the skill menu, accessible via the `#` command,
-which summarizes all the stats conferred by organs and conditions listed
-in the organ menu, invoked by `@`.
+The current pointman's melee bonus (`xdy + %`, which means rolling 
+`x`-many `y`-sided dice + some bonus values), armor modifier 
+(`[melee]` and `{ranged}` bonuses) and other detailed stats can be viewed
+in the skill menu, accessible via the `#` command, which summarizes 
+all the stats conferred by organs and conditions listed in the organ menu,
+invoked by `@`.
+It is always good to check the condition of your character, as you may notice
+some important changes in the character's condition. It would be very sad
+to lose control over your teammate just due to not finding out that he/she
+was terrified by some monstrosity that threatened and impressed him/her
+some time ago.
+The same information about enemies may be found by the `~` command while
+the enemy unit is in the crosshair. For example, you see an unknown creature.
+How to react to it? Is it dangerous? Check its lore menu and the organs 
+it has! It may help you to generate the appropriate strategy. For example, 
```

Comment:
I guess "come up" or "create" or "invent" is more natural than "generate"

Reply:
changed